org-formation/organization.yaml

```diff
@@ -288,6 +289,20 @@ Organization:
         budget-alarm-threshold: 2000
         budget-alarm-threshold-email-recipient: aws-iatlas-prod@sagebase.org
 
+  # Provisioner repository: https://github.com/ncihtan/aws-htan-infra
+  HtanDevAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: htan-dev
```

Comment:
convention for account name seems to be CamelCase.

Reply:
There's a mix in this file. I've basically been taking the value for `Alias` minus the `org-sagebase-` prefix.